vmdb/app/models/miq_bulk_import.rb

```diff
@@ -52,7 +52,7 @@ def self.upload(fd, tags, keys)
   def self.find_entry_by_keys(klass, keys)
     keys2array = keys
     primary_key, primary_key_value = keys2array.shift
-    result = klass.all(:conditions => "LOWER(#{primary_key}) LIKE '#{primary_key_value.downcase}'")
+    result = klass.where("LOWER(#{primary_key}) LIKE '#{primary_key_value.downcase}'")
     return result if result.empty? || result.length == 1
```

Comment:
Will this cause two queries or does the `.empty?` deal with caching?  Perhaps the code should just do `.size <= 1` to be safe.